sigs/sig-serverless.md

```diff
@@ -0,0 +1,139 @@
+complexities of the platform hosting/managing their applications. As such,
+this SIG will focus on technologies related to enabling this simplified
+developer experience.
+
+## Areas Considered In Scope
+
+Technologies related to hosting, managing, developing and integrating of
+Serverless applications from an application developer's perspective.
+The following are example areas that are considered in scope:
+
+* container hosting platforms that attempt to abstract their complexities away
+  from the application developer
+* orchestration and management of workloads running on those platforms from the
+  application developer's perspective
+* tooling to aid in the development and deployment of those workloads
+* integration technologies that aim to promote interoperability between the
+  various systems that an application developer may interact with during the
+  management of their applications on those platforms
+* workload interoperability - e.g. portability of functions,
+  interaction with hosting environment/runtime
+* interoperability and connectivity with backend services used by worloads
+
+## Areas Considered Out Of Scope
+
+Anything not considered in scope above is out of scope.  See also “Interfaces
+with Related Groups” below.
+
+An example would include technology related to the inner-workings of a
+particular platform. For example, a new autoscaler for Kubernetes.
+
+## SIG Mission Statement
+
+To enable application developers to more easily create, deploy and manage their
+applications without the need to become experts in the hosting technologies
+being used.
+
+Developers should focus their attention on the development of their
+applications. When they are required to spend a significant portion of their
+time learning and interacting with the hosting platform of those applications
+then that is time that should have been spent working on their business
+objective.
+
+To that end, the SIG will:
+
+1. Provide valuable and unbiased information to the TOC,
+   End Users and Projects of the CNCF regarding areas considered in scope
+   (see above).
+2. Collaborate effectively with other related groups (see below).
+3. Help to maintain the continued health of the CNCF Projects deemed
+   to be within the scope of this SIG (see below)
+4. Identify and fill gaps in the landscape of CNCF Projects within scope
+   either through encouragement of existing projects to join the CNCF
+   or through development of new artifacts (e.g. specs, code, papers).
+5. Maintain any "living" documentation, such as landscape and state of the
+   community documents.
+
+Specific SIG deliverables are as per the above, and the [general SIG
+responsibilities set out by the CNCF
+ TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+## Current CNCF Projects Considered To Be Within The Scope Of This SIG
+
+* CloudEvents
+* Serverless Workflow
+* CloudEvents Discovery API
+* CloudEvents Subscription API
+
```

Comment:
One might argue that KEDA has links to SIG Runtime & SIG Serverless. What do you think @raravena80?
```suggestion
* Kubernetes Event-driven Autoscaling (KEDA)

```

Or do we have to belong to only one SIG?

Reply:
Sounds fair to me!